src/System.Xml.XmlSerializer/tests/XmlSerializerTests.cs

```diff
@@ -593,7 +593,7 @@ public static void Xml_Enums()
     public static void Xml_Nullables()
     {
         var item = new WithNullables() { Optional = IntEnum.Option1, OptionalInt = 42, Struct1 = new SomeStruct { A = 1, B = 2 } };
-        var actual = SerializeAndDeserialize(item, "<?xml version=\"1.0\"?>\r\n<WithNullables xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns:xsd=\"http://www.w3.org/2001/XMLSchema\">\r\n  <Optional>Option1</Optional>\r\n  <Optionull xsi:nil=\"true\" />\r\n  <OptionalInt>42</OptionalInt>\r\n  <OptionullInt xsi:nil=\"true\" />\r\n  <Struct1>\r\n    <A>1</A>\r\n    <B>2</B>\r\n  </Struct1>\r\n  <Struct2 xsi:nil=\"true\" />\r\n</WithNullables>");
+        var actual = SerializeAndDeserialize(item, "<?xml version=\"1.0\"?>" + Environment.NewLine + "<WithNullables xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns:xsd=\"http://www.w3.org/2001/XMLSchema\">\r\n  <Optional>Option1</Optional>\r\n  <Optionull xsi:nil=\"true\" />\r\n  <OptionalInt>42</OptionalInt>\r\n  <OptionullInt xsi:nil=\"true\" />\r\n  <Struct1>\r\n    <A>1</A>\r\n    <B>2</B>\r\n  </Struct1>\r\n  <Struct2 xsi:nil=\"true\" />" + Environment.NewLine + "</WithNullables>");
```

Comment:
Good catch.  I found a bunch of these in this file, fixed them, and fixed all of the formatting to make it read more nicely.